pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/loaders/ReportLoader.kt

```diff
@@ -3,8 +3,15 @@ package gov.cdc.ocio.processingstatusapi.loaders
 import com.azure.cosmos.models.CosmosQueryRequestOptions
 import gov.cdc.ocio.processingstatusapi.models.Report
 import gov.cdc.ocio.processingstatusapi.models.dao.ReportDao
+import gov.cdc.ocio.processingstatusapi.models.DataStream
+import graphql.schema.DataFetchingEnvironment
+import io.ktor.server.auth.*
+import io.ktor.server.auth.jwt.*
 import java.time.ZoneOffset
 
+class ForbiddenException(message: String) : RuntimeException(message)
+
+
 class ReportLoader: CosmosLoader() {
 
     fun getAnyReport(): Report? {
```

Comment:
Do we need this function

Reply:
Yes, we use the ForbiddenException inside CustomGraphQLStatusPages.kt.  It gets thrown when the use is authenticated but runs into an authorization issue.